### DISCUSSION TITLE
Issues with user setup / initial_admin_keys

### DISCUSSION THREAD

**Discussion by CptPotato** [charmbracelet/soft-serve#539] [Q&A]
I've been trying to set up a soft-serve docker container. Accessing the server cli interface works fine but I'm unable to set up any users or configure the server with the `settings` command, getting `Error: unauthorized` every time.

I have set `initial_admin_keys` to my ed25519 key and also tried setting the `INITIAL_ADMIN_KEYS` env var in `compose.yml`, but nothing seems to work. Anyone know what might be the issue?

**Comment by catbash** [comment]
I was also running into this issue at first, but setting `SOFT_SERVE_INITIAL_ADMIN_KEYS` directly (i.e., using the environment variable, not using `compose.yml`) _before_ running `soft serve` for the first time fixed it in my case.
